Bottom: 953bbd3ae246d61b524a3f19679eb047e201d97a
Top:    a1838f2f9984ffdcd3432d0746682a20218a7077
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-02 17:38:39 +0000

Batch size for accumulating lines down to 1

This works around a bug seen on one of my boards, and is similar
to what I had in the non-pours variant of the code.


---

diff --git a/src/pour.c b/src/pour.c
index d1099cd..d9ee86f 100644
--- a/src/pour.c
+++ b/src/pour.c
@@ -68,7 +68,7 @@ RCSID ("$Id$");
 
 #define UNSUBTRACT_BLOAT 100
 #define SUBTRACT_PIN_VIA_BATCH_SIZE 100
-#define SUBTRACT_LINE_BATCH_SIZE 20
+#define SUBTRACT_LINE_BATCH_SIZE 1
 
 /* ---------------------------------------------------------------------------
  * local prototypes
